contribution/models.py

```diff
@@ -45,6 +47,7 @@ class Premium(core_models.VersionedModel):
     )
     reporting_id = models.IntegerField(db_column="ReportingId", blank=True, null=True)
     audit_user_id = models.IntegerField(db_column="AuditUserID")
+    created_date = models.DateTimeField(db_column="CreatedDate", default=datetime.now())
```

Comment:
Should we create migration reflecting this change?

Reply:
Nope, the field is added in legacy database script.

Reply:
I fixed the default value factory